lossfunctions_ag.bib

```diff
@@ -0,0 +1,75 @@
+@InProceedings{pmlr-v32-yanga14,
+  title = 	 {The Coherent Loss Function for Classification},
+  author = 	 {Wenzhuo Yang and Melvyn Sim and Huan Xu},
+  booktitle = 	 {Proceedings of the 31st International Conference on Machine Learning},
+  pages = 	 {37--45},
+  year = 	 {2014},
+  editor = 	 {Eric P. Xing and Tony Jebara},
+  volume = 	 {32},
+  number =       {1},
+  series = 	 {Proceedings of Machine Learning Research},
+  address = 	 {Bejing, China},
+  month = 	 {22--24 Jun},
+  publisher = 	 {PMLR},
+ 
+}
+@article{vapnik1998statistical,
+  title={Statistical learning theory},
+  author={Vapnik Vladimir, N},
+  journal={New York: W iley, l998},
+  year={1998}
+}
+
+@book{Davidson-Pilon:2015:BMH:2851115,
+ author = {Davidson-Pilon, Cameron},
+ title = {Bayesian Methods for Hackers: Probabilistic Programming and Bayesian Inference},
+ year = {2015},
+ isbn = {0133902838, 9780133902839},
+ edition = {1st},
+ publisher = {Addison-Wesley Professional},
+}
+
+@book{Scholkopf:2001:LKS:559923,
+ author = {Scholkopf, Bernhard and Smola, Alexander J.},
+ title = {Learning with Kernels: Support Vector Machines, Regularization, Optimization, and Beyond},
+ year = {2001},
+ isbn = {0262194759},
+ publisher = {MIT Press},
+ address = {Cambridge, MA, USA},
+} 
+ 
+@article{DBLP:journals/corr/JanochaC17,
+  author    = {Katarzyna Janocha and
+               Wojciech Marian Czarnecki},
+  title     = {On Loss Functions for Deep Neural Networks in Classification},
+  journal   = {CoRR},
+  volume    = {abs/1702.05659},
+  year      = {2017},
+  url       = {http://arxiv.org/abs/1702.05659},
+  archivePrefix = {arXiv},
+  eprint    = {1702.05659},
+  timestamp = {Mon, 13 Aug 2018 16:46:10 +0200},
+  biburl    = {https://dblp.org/rec/bib/journals/corr/JanochaC17},
```

Comment:
Read the guidelines and remove all these useless fields.  url, eprint, timestamp, biburl etc.

Reply:
ok replaced bib file